CHANGELOG.md

```diff
@@ -2,9 +2,20 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
-### master
```

Comment:
Really delete this? Or do we leave them for the next round of changes?

Reply:
Either way is fine. I'll add it back in so that others don't have to remember.